### PR TITLE
Repath janicarts/secways/atvs

### PR DESCRIPTION
**Pull request by neersighted**
Remove the path compability in favor of true pathing

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Who doesn't love a good structured bed chair vehicle? :stuck_out_tongue:

File: code/__PATH_COMPATIBILITY/path_compatibility.dm [hunk 1]
```diff
@@ -1,28 +0,0 @@
-/*
```
Comment (Iamgoofball) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: Nope, I'm using this for my ANIMAL FARM pr.

Please keep the file.

File: code/__PATH_COMPATIBILITY/path_compatibility.dm [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
-	updating old maps.
-
-	tl;dr TYPEFUCKERY, because fuck updating all these maps
-
-*/
-
-
-
-//Vehicle Refactor - 2015
-/obj/structure/bed/chair/janicart/secway
```
Comment (Iamgoofball) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: Plus, these should be left in for downstreams with unique maps that may not be using the new typepaths but want to update.

File: code/__PATH_COMPATIBILITY/path_compatibility.dm (hunk 1 above)
Comment (RemieRichards) [inline code/__PATH_COMPATIBILITY/path_compatibility.dm]: He's keeping the folder+file, I told him to, He's just taking his time to do it.

**Comment by neersighted** [comment]
Yeah just a little indisposed, give me an hour

**Pull request merged**